<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdj">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Матрица Смежности</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>500</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddVertex">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить Вершину</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddEdge">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>100</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить Ребро</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>240</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить Элемент</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>180</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSW">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>330</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Дийкстра</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>390</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <zorder>spinBox_2</zorder>
  <zorder>graphicsView</zorder>
  <zorder>btnAdj</zorder>
  <zorder>label</zorder>
  <zorder>btnAddVertex</zorder>
  <zorder>btnAddEdge</zorder>
  <zorder>btnDelete</zorder>
  <zorder>spinBox</zorder>
  <zorder>btnSW</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
